main.go

```diff
@@ -338,8 +341,22 @@ func getSetIPTablesRulesServerChainElement() networkservice.NetworkServiceServer
 		"-A NSM_POSTROUTING -j SNAT --to-source {{ index .NsmDstIPs 0 }}",
 		"-A POSTROUTING -p tcp -o {{ .NsmInterfaceName }} -j NSM_POSTROUTING",
 	}
+	cfg, err := ioutil.ReadFile(filepath.Clean(path))
+	if err != nil {
+		log.FromContext(ctx).Error(err)
+		return defaultRules
+	}
+	var rules []string
+	err2 := yaml.Unmarshal(cfg, &rules)
+	if err2 != nil {
+		log.FromContext(ctx).Error(err2)
```

Comment:
Please add some message in logs, it could simplify bug fixiging.

```suggestion
		log.FromContext(ctx).Errorf("TODO: %v",err)
```